cypress/plugins/index.js

```diff
@@ -22,7 +23,11 @@ module.exports = function(on, config) {
   addMatchImageSnapshotPlugin(on, config);
   on('before:browser:launch', function(browser, args) {
     if (browser.name === 'chrome') {
-      args.push('--window-size=1280,1200');
+      if (os.platform() === 'win32') {
+        args.push('--window-size=1295,1200');
+      } else {
+        args.push('--window-size=1280,1200');
+      }
```

Comment:
What's the reason for the difference here?

Reply:
I'm not sure.

Reply:
I would prefer that we use [cy.viewport](https://docs.cypress.io/api/commands/viewport.html#Syntax) or [set it in config](https://docs.cypress.io/guides/references/configuration.html#Viewport). That should be the most consistent method. This appears to be taking the window skin into account, and that will not be consistent if the user has customized their theme.

Reply:
`cypress-image-snapshot` doesn't respect `cy.viewport`.  I'm fairly certain I tried to get this to work initially this way as it would be a cleaner approach.  If you see a way to do this, let me know.

The config is static and already set.  I'm not sure how changes here could be dynamic.

Also, I think (but have not tested) Cypress' instance of Chrome might not be affected by user changes to window skin.  It's definitely a modified version of Chrome that doesn't seem to pickup changes from the native instance.

![image](https://user-images.githubusercontent.com/44378016/63448257-cf612000-c3fa-11e9-92de-f8b87c9b23ed.png)